client/components/deposits-status/index.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { Icon } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+
+const DepositsStatus = ( props ) => {
```

Comment:
What is the difference between this component and `PaymentsStatus`? It looks like they're doing the same thing. If that's the case, what do you think about renaming the `PaymentsStatus` to a more generic name (like `SectionStatus`) and using that for both rows?

Reply:
Good idea - they displayed different things in WCPay but trimming them down made them essentially the same. 
They're now a common SectionStatus component 👍